stanford-earth/SE3_BLT
# EARTH-000: Added pathauto default configurations for nodes and users.

# READY FOR REVIEW

# Summary
- Added default pathauto patterns for users and nodes. Individual node type patterns will override default path.

# Needed By (Date)
- when you can

# Urgency
- low urgency

# Steps to Test

1. Checkout branch & sync configs.
2. create a node/user
3. the path alias should contain the node title or the user name respectively.
